datasource/jaxb/pom.xml

```diff
@@ -57,7 +57,7 @@
                         </goals>
                         <configuration>
                             <sources>
-                                <source>${project.build.directory}/generated-sources</source>
```

Comment:
Eclipse M2E plugin can't handle this. 
The alternate works for Eclipse and the MVN cli